flink-cdc-composer/src/main/java/org/apache/flink/cdc/composer/flink/deployment/YarnApplicationDeploymentExecutor.java

```diff
@@ -121,6 +122,10 @@ public PipelineExecution.ExecutionInfo deploy(
 
     private String getFlinkCDCDistJarFromEnv() throws IOException {
         String flinkCDCHomeFromEnvVar = System.getenv(FLINK_CDC_HOME_ENV_VAR);
+        Preconditions.checkNotNull(
+                flinkCDCHomeFromEnvVar,
+                "FLINK_CDC_HOME is not correctly set in environment variable, current FLINK_CDC_HOME is: "
+                        + FLINK_CDC_HOME_ENV_VAR);
         Path flinkCDCLibPath = new Path(flinkCDCHomeFromEnvVar, "lib");
```

Comment:
Current FLINK_CDC_HOME is derived from flink-cdc.sh. When users integrate flink-cdc, they do not only submit jobs through flink-sh.sh.